index.html

```diff
@@ -0,0 +1,174 @@
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+    <div class="container">
+        <div class="image">
+            <img class="img" src="./images/multi_2.png" alt="">
+        </div>
+        <div class="content">
+            <h2 class="content-header">Multi-Post <br>
+              Stories</h2>
+            <ul class="specification">
+              <li class="name">Aliza</li>
+              <li class="skills"><img src="./images/dot.png" alt="">
+              <p> Back End Dev</p></li> 
+              <li class="year"><img src="./images/dot.png" alt="">
+                <p>2015</p></li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+  </section>
+<section class="about-container">
+
+  <div class="about">
+<h2 class="about-heading">
+  About Myself
+</h2>
+<p class="about-para">
+  Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h4 class="connect">LET'S CONNECT</h4>
+<ul class="about-icons">
+  <li class="twitter"><a href=""><img src="./images/twitter.png" ></a> </li>
+  <li class="linkedln"><a href=""><img src="./images/Linkedin.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/gmail.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/github.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/Vector.png" alt=""></a></li>
+
+</ul>
+<button class="get-resume">get my resume</button>
+  </div>
+<div >
+  <div class="language-section" >
+    <p class="languages">Languages </p>
+    <img class="arrow" src="./images/downarrow.png" alt="">
+  </div>
+    <ul class="lang-name">
+      <li class="javascript-lang"> <img src="./images/js.png" alt=""> Javascript</li>
+      <li class="javascript-lang"><img src="./images/html.png" alt="">Html</li>
+      <li class="javascript-lang"><img src="./images/css.png" alt="">Css</li>
+      
+  </ul>
+</div>
+<ul class="after-about">
+  <li class="frameworks">Frameworks <img class="sidearrow1" src="./images/arrow.png" alt=""></li>
+  <img src="./images/sep.png" alt="">
+  <li class="frameworks">Skills <img class="sidearrow2" src="./images/arrow.png" alt=""></li>
+  <img src="./images/sep.png" alt="">
+</ul>
```

Comment:
As requested by the previous code reviewer kindly follow these instructions to meet the project requirements:-

Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with a detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
      </li>
      <li>
        Skills
      </li>
    </ul>
```